Contributing.md

```diff
@@ -159,6 +159,12 @@ private:
 - Empty-bodied loops should use braces after their header, not a semicolon.
   - Yes: `while (condition) {}`
   - No: `while (condition);`
+- For do-while loops, place 'while' on the same line as the closing brackets
+  ```c++
```

Comment:
Put a new line in between the sentence and the code block, otherwise it'll be formatted wrong.